main/FatesHistoryInterfaceMod.F90

```diff
@@ -2922,6 +2927,10 @@ subroutine update_history_dyn1(this,nc,nsites,sites,bc_in)
             hio_lai_secondary_si(io_si) = 0._r8
          end if
 
+         ! Normalize crown-area weighted height
+         if(site_ca>nearzero)then
+            hio_ca_weighted_height_si(io_si) = hio_ca_weighted_height_si(io_si)/site_ca
+         end if
```

Comment:
BA weighted height is set to zero if site_ba is not > nearzero. Seems like it would be a very small number anyway but do we want the same thing for CA weighted height? 

Also the comment on 2867 could be updated to say that ca_weighted_height and ba_weighted_height are normalized outside of the patch loop.

Reply:
I left it out because we zero everything in an automated function anyway, we don't actually need to zero the ba weighted output.  But now that you bring the point up, its probably confusing to anyone reading the code why we do things differently. I think its better to have these outputs processed consistently to avoid confusion. I'll do a look through and see if I can make things more consistent in other places as well.

Reply:
Thanks, that makes sense.